### PR TITLE
fix: deeplink when app not open on linux

### PR DESCRIPTION
**Pull request by namchuai**
## Describe Your Changes

- Fix deep link for linux user

## Example
Deeplink: `jan://models/huggingface/TheBloke/Magicoder-S-DS-6.7B-GGUF`

## Self Checklist

- [ ] Added relevant comments, esp in complex areas
- [ ] Updated docs (for bug fixes / features)
- [ ] Created issues for follow-up changes or refactoring needed

### CODE REVIEW

**Comment by Van-QA** [comment]
#2876

**Review by Van-QA — APPROVED** [review]
LGTM

File: web/containers/Providers/DeepLinkListener.tsx [hunk 1]
```diff
@@ -27,46 +27,79 @@ const DeepLinkListener: React.FC<Props> = ({ children }) => {
...
+        setLoadingInfo(undefined)
+        let errMessage = 'Unexpected Error'
+        if (err instanceof Error) {
+          errMessage = err.message
+        }
+        toaster({
+          title: 'Failed to get Hugging Face models',
+          description: errMessage,
+          type: 'error',
+        })
```
Comment (louis-jan) [inline web/containers/Providers/DeepLinkListener.tsx:61]:
```suggestion
        toaster({
            title: 'Failed to get Hugging Face models',
            description: err instanceof Error ? err.message : 'Unexpected Error',
            type: 'error',
        })
```

**Review by louis-jan — APPROVED** [review]
LGTM

**Pull request merged**